isisd/isis_nb_config.c

```diff
@@ -213,6 +214,9 @@ int isis_instance_area_address_destroy(struct nb_cb_destroy_args *args)
 	uint8_t buff[255];
 	struct isis_area *area;
 	const char *net_title;
+	struct listnode *cnode;
+	struct isis_circuit *circuit;
+	int lvl;
```

Comment:
nit: it would be nice to move these variables into a smaller scope to improve code readability (non-blocking comment).